e2e/cephfs.go

```diff
@@ -2478,6 +2478,19 @@ var _ = Describe(cephfsType, func() {
 				}
 			})
 
+			By("test volumeGroupSnapshot", func() {
+				scName := "csi-cephfs-sc"
```

Comment:
I don't see StorageClass being creating in this test case. This still works because the previous test case has created it and didn't delete it after completion of test case.

Reply:
Yes it needs a cleanup which am working on, it will be a separate PR